packages/dart/sshnp_flutter/lib/src/presentation/widgets/home_screen_widgets/home_screen_core.dart

```diff
@@ -52,7 +55,52 @@ class _HomeScreenCoreState extends ConsumerState<HomeScreenCore> {
       },
       data: (profiles) {
         if (profiles.isEmpty) {
-          return const Text('No SSHNP Configurations Found');
+          return Center(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                gapH40,
+                Text(
+                  'Get Started!',
+                  style: Theme.of(context).textTheme.bodyMedium!.copyWith(color: kHomeScreenGreyText),
+                ),
+                gapH16,
+                GestureDetector(
+                  onTap: () => HomeScreenActionCallbacks.newProfileAction(ref, context),
+                  child: Stack(
+                    children: [
+                      // const NewProfileAction(),
+                      SvgPicture.asset(
+                        'assets/images/combo.svg',
```

Comment:
The file name checked into git is uppercase where as we use lowercase here. Also, I couldn't find this asset in pubspec.yaml.

Reply:
The case doesn't seem to matter since the app renders the svg. I renamed it but the change isn't available in git to add.

Reply:
I removed unused asset images.